log/kafka/kafka.go

```diff
@@ -1,179 +0,0 @@
-/*
```

Comment:
That's totally gone? No kafka logging support out of the box anymore?

Reply:
as long as we don't need it ... we can always bring it back